src/openrct2/paint/Supports.cpp

```diff
@@ -473,10 +521,15 @@ bool wooden_a_supports_paint_setup(
     {
         special = (special - 1) & 0xFFFF;
 
-        int32_t imageId = WoodenCurveSupportImageIds[supportType];
+        int32_t imageId;
+        if (special <= 48)
+            imageId = WoodenCurveSupportImageIds[supportType];
+        else
+            imageId = WoodenCurveSupportImageIdsG2[supportType];
+
```

Comment:
Could you explain what you did in this file?
And what is the meaning of 48/49?

Reply:
The original code assumes that all the support sprites have contiguous sprite IDs. But since the old sprites are in g1.dat and the sprites I added are in g2.dat, they cannot be contiguous. So I added a separate array to hold the additional sprites. 48 is, IIRC, the number of original support sprites - the idea is that support sprite number 49 maps to support sprite 0 in the g2.dat array. Since there's no mechanism to add new sprites to g1.dat, and even if there was it would change the locations of every subsequent sprite, I don't think there's a way around this even if it's a bit dirty.

Reply:
Right, but what kind of support sprites have you added?

Reply:
I added support sprites for large flat to steep and steep to flat transitions. I did this because the base game doesn't have support sprites for those track pieces but I wanted them to be available on this one. The sprites I added are in resources/g2/support.

Reply:
It would have been better to change it so that instead of using contiguous sprite id's it used a table lookup. That would prevent the additional branch and allow for future remapping. Leave as is for now but keep that in mind in future.